### ISSUE TITLE
Compilation error under linux

### ISSUE THREAD

**Issue by magiblot** [vapoursynth/vapoursynth#359]
I'm trying to compile the source code obtained via
> git clone https://github.com/vapoursynth/vapoursynth.git

(as suggested in the documentation)

This makes no sense... (terminal output of ./configure)

```
checking for a Python interpreter with version >= 3... python3
checking for python3... /usr/bin/python3
checking for python3 version... 3.4
checking for python3 platform... linux
checking for python3 script directory... ${prefix}/lib/python3.4/site-packages
checking for python3 extension module directory... ${exec_prefix}/lib64/python3.4/site-packages
checking for PYTHON3... no
configure: error: Package requirements (python-3.4) were not met:

No package 'python-3.4' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables PYTHON3_CFLAGS
and PYTHON3_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```
(It looks for a 'python-3.4' package in spite of the actual python 3.4 being installed and detected).

I guess this is a bug in the installer, but if you need any information about my system and installed packages I will provide it.

**Comment by magiblot** [comment]
Since I'm experiencing the same issue with the latest official releases, I'm enclosing the complete output of autogen and configure.
My OS is OpenSUSE Leap 42.2 x86_64.

[compilation_error.txt](https://github.com/vapoursynth/vapoursynth/files/1648985/compilation_error.txt)

**Comment by myrsloik** [comment]
For some reason the relevant pkg config file can't be found...
Did you install python using yast?  (or whatever the  package manager is called on suse)

**Comment by dubhater** [comment]
Perhaps you don't have the ``python3-devel`` package?

**Comment by magiblot** [comment]
Hi there, thank you very much for your attention.

That's it! Although I had many python3 packages installed, devel was not one of them.

I suggest adding a reminder for the devel packages [in the documentation](http://www.vapoursynth.com/doc/installation.html#linux-installation-from-packages). This is my first time installing a python3 project and I was missing it :P

**Comment by myrsloik** [comment]
The short and not so nice answer is that basic skills in compiling are assumed. Including handling your package manager of choice. If you can make a specific list of which packages opensuse needs (as in works from a clean installation) I'll add it to the list as a hint.

**Comment by magiblot** [comment]
From what I can see, openSUSE offers several packages for things other linux distributions just offer one (for example, there are no `-devel` packages in ArchLinux because these features are already in the main package). Although I'm not a developer I have compiled other projects from source and I've had to install their missing dependencies. I could usually find the missing packages' names either in the documentation or in the `configure` logs, but this time `PYTHON3`/`python-3.4` was not enough for me to figure out what the missing package was.
It was just a coincidence of inexperience and bad luck: it's not your fault at all. I can't make a detailed list of package dependencies for OpenSUSE. However, now that we are aware that some linux distributions require the additional separate package `python3-devel` it would seem appropiate to me to add a note about this, at least at the point of `configure` where I experienced the error.
